cloud-director/v30.1.1/release.yaml

```diff
@@ -0,0 +1,112 @@
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  name: cloud-director-30.1.1
+spec:
+  apps:
+  - name: capi-node-labeler
+    version: 1.0.2
+  - name: cert-exporter
+    version: 2.9.5
+    dependsOn:
+    - kyverno-crds
+  - name: cert-manager
+    version: 3.9.0
+    dependsOn:
+    - prometheus-operator-crd
+  - name: chart-operator-extensions
+    version: 1.1.2
+    dependsOn:
+    - prometheus-operator-crd
+  - name: cilium
+    version: 0.31.1
```

Comment:
Could you bump cilium to the latest 0.31.x version?

Reply:
So I need to create new release of AWS first? How does the workflow works? 

We'd like to ship the latest 0.31.x version of cilium in the latest v30.1 patch.

Reply:
> We agreed on doing minor releases on a monthly basis lately. 

What that announced somewhere?

> These will definitely include the latest patch version of Cilium.

That's not what I'm asking in this PR. The next minor release should include the latest version of cilium for sure. Here I'm asking to include a patch release of an older version.

> Is there an urgent reason to update Cilium or is it mostly on a "please bump as you go" basis?

It's not about urgency but rather being able to ship the work we did already and prepared.

Reply:
This particular release is meant to ship a hot fix for the v30.1.0 release as the latter is currently not useable for customers.

We can safely bump Cilium to the latest patch release in another release. But as there's a single purpose for this release and we already released a CAPA v30.1.1 with just the Security Bundle patch, we won't include any more changes for the sake of keeping releases aligned across providers.

I'll bring up this topic on Monday and try to come up with a round of minor releases next week!

Reply:
Also
> We agreed on doing minor releases on a monthly basis lately.

is not official or announced as those [changes](https://github.com/giantswarm/roadmap/issues/3760) in the process will most likely come with v31.